web_ui/frontend/app/director/page.tsx

```diff
@@ -63,7 +48,7 @@ export default function Home() {
                 <Grid item xs={12} lg={8} xl={6}>
                     <Typography variant={"h4"} pb={2}>Origins</Typography>
                     {originData ?
```

Comment:
If the filter returns 0 entries, I got the skeleton animation:
![Screenshot 2024-06-11 at 10 05 21 AM](https://github.com/PelicanPlatform/pelican/assets/41393704/9c3b17ea-f782-4820-9070-3d7133687c5b)
While I think the user is expecting an empty list with text suggesting no result found